examples/ekfutils/postprocessing.py

```diff
@@ -0,0 +1,92 @@
+import os
+import numpy as np
+import matplotlib.pyplot as plt
+
+from pymlg import SE3
+
+class History:
+    def __init__(self):
+        self.timestamps = np.empty(0)
+        self.states = np.empty((0, 4, 4))
+        self.covariances = np.empty((0, 6, 6))
+
+    def add(self, timestamp: float, state: np.ndarray, covariance: np.ndarray):
+        self.timestamps = np.append(self.timestamps, timestamp)
+        self.states = np.append(self.states, state.reshape(1, 4, 4), axis=0)
+        self.covariances = np.append(self.covariances, covariance.reshape(1, 6, 6), axis=0)
+
+    def get(self):
+        return self.timestamps, self.states, self.covariances
+    
+class Evaluate:
```

Comment:
Same comment as History. I think "Evaluate" should be changed for something like "Results," like in navlie